src/chigame/api/serializers.py

```diff
@@ -1,9 +1,26 @@
 from rest_framework import serializers
 
-from chigame.games.models import Game
+from chigame.games.models import Game, Lobby
 
 
 class GameSerializer(serializers.ModelSerializer):
     class Meta:
         model = Game
         fields = ("id", "name", "description", "min_players", "max_players")
+
+
+class LobbySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lobby
+        fields = (
+            "id",
+            "name",
+            "game",
+            "game_mod_status",
+            "created_by",
+            "members",
+            "min_players",
+            "max_players",
+            "time_constraint",
+            "lobby_created",
```

Comment:
Very good generalizable formatting for listing the specific fields.
I recommend you do line separation like this in general when you have lists of items.

Reply:
I merged the `dev` branch into `apis/lobby-endpoints` so all of the merge conflicts should be fixed.